### PR TITLE
Bring back Minent for segmentation head

### PR DESCRIPTION
**Pull request by alexrey88**
Small modif to use minimization of entropy for segmentation head.
Diff experiment: [here](https://www.comet.ml/alexrey88/omnigan/8b4d029579ce4a46b28808f50f26568a/e4464d71cdc24a0eaf7b2eedacc76ba7/compare?experiment-tab=params)

### CODE REVIEW

File: omnigan/trainer.py [hunk 1]
```diff
@@ -214,9 +214,7 @@ def resume_from_path(
         if new_exp:
             exp = Experiment(project_name="omnigan", **comet_kwargs)
             exp.log_asset_folder(
-                str(Path(__file__).parent),
-                recursive=True,
-                log_file_name=True,
+                str(Path(__file__).parent), recursive=True, log_file_name=True,
```
Comment (vict0rsch) [inline omnigan/trainer.py:217]: what's going on here? Thant's what I have on master
2020-11-04 at 19 19

Comment (vict0rsch) [reply omnigan/trainer.py:217]: Why does it say masster has log_asset_folder on multiple lines?

**Review by vict0rsch — APPROVED** [review]

File: omnigan/trainer.py (hunk 1 above)
Comment (alexrey88) [reply omnigan/trainer.py:217]: weird indeed.. not sure why it did that

Comment (alexrey88) [reply omnigan/trainer.py:217]: Thought it was just black that reformatted the line..

Comment (vict0rsch) [reply omnigan/trainer.py:217]: Yeah, anyway, doesn't really matter since the right version is in master

**Pull request merged**